Palindrome.UnitTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+// Added a test for Empty Input.
+// Created branch 'review' for PR sharing.
+
+
+namespace PalindromeArray.UnitTests
+{
+    [TestClass]
+    public class PalindromeUnitTests
+    {
+        [TestMethod]
+        public void Check_Empty_Input()
+        {
+            // Arrange
+            Palindrome check = new Palindrome();
+            char[] arrToCheck = check.PalindromeHelper("");
```

Comment:
This is a general comment about the helper. It seems like the `PalindromeHelper` method is only being used from tests, not from production code, so you should move it into the test class itself.

Reply:
True, I'll move it to test class. Sloppy design.